en/git-commit.txt

```diff
@@ -307,7 +307,7 @@ The possible options are:
 	- 'normal' - Shows untracked files and directories
 	- 'all'    - Also shows individual files in untracked directories.
 
-The default can be changed using the status.showUntrackedFiles
+The default can be changed using the `status.showUntrackedFiles`
```

Comment:
Not sure about this one. The markup is not consistant, but it seems configuration variables are not back-quoted, when commands and command line options are.

Reply:
Yeah right